README-fi.md

```diff
@@ -10,7 +10,7 @@ API ja optimointi modi [3Dash](https://delugedrop.itch.io/3dash):ille
 - Muita ominaisuuksia jotka voivat auttaa 3Dash modaajia
 
 ## Asennus
-1. Lataa BepInEx asennusohjelma [script](https://gist.githubusercontent.com/cgytrus/29085a6bf179893666316a36e1c92bf6/raw/bepinex-installer.ps1)
+1. Lataa BepInEx asennusohjelma. [Scripti](https://gist.githubusercontent.com/cgytrus/29085a6bf179893666316a36e1c92bf6/raw/bepinex-installer.ps1)
```

Comment:
i don't think the sentence is supposed to end here
also not translated on line 16 :p
you said it's hard to translate so if you think people will understand it, it's better to leave it in english

Reply:
It should be fully translated, I tried to make it as clear as possible to read.